.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,34 @@
+name: Node CI
+on: [push]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        browser: [ChromeHeadless, FirefoxHeadless]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+    - name: Checking out Repository
+      uses: actions/checkout@v1
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Install Chrome (macOS)
+      if: matrix.os == 'macOS-latest' && matrix.browser == 'ChromeHeadless'
+      run: brew cask install google-chrome
+    - name: Install Firefox (macOS)
+      if: matrix.os == 'macOS-latest' && matrix.browser == 'FirefoxHeadless'
+      run: brew cask install firefox
+    - name: Install Dependencies
+      run: yarn bootstrap
+    - name: Run Tests
+      run: yarn test
+      env:
+        BROWSER: ${{ matrix.browser }}
+    - name: Upload Test Coverage to codecov
+      # Only upload it a single time
+      if: matrix.os == 'ubuntu-latest' && matrix.browser == 'ChromeHeadless'
+      run: yarn run codecov
```

Comment:
Codecov can handle receiving multiple sets of coverage results, though. This actually comes in handy for places where different browsers hit different branches.